Lib/test/test_symtable.py

```diff
@@ -205,12 +205,14 @@ def test_assigned(self):
 
     def test_annotated(self):
         st1 = symtable.symtable('def f():\n    x: int\n', 'test', 'exec')
-        st2 = st1.get_children()[0]
+        st2 = st1.get_children()[1]
```

Comment:
It was a bit odd to me at first that for classes, their `__annotate__` symbol table shows up as a child of the class symbol table, but for functions, it shows up as a sibling of the function symbol table.

But I guess on further reflection this makes some sense. `__annotate__` is in fact part of the class namespace/scope. For functions, it is an attribute of the function object, but not actually part of the function scope, which is the namespace of the function body. So I guess a sibling scope is the most reasonable representation.